packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -16,7 +17,7 @@ const SideNotes = ({ side_notes }) => {
     );
 };
 
-const Content = ({ is_routed, items, selected }) => {
+const Content = ({ is_routed, items, selected, is_dashboard }) => {
```

Comment:
We can use `PlatformContext` here.

Reply:
Should we even consider putting `context` within components? The atomic / UI components should be as agnostic from the packages as possible. 

If we want to have any style overrides, we can handle those with another wrapper component that imports the `VerticalTab`  in the package where the `PlatformContext` is referenced to.

Reply:
Thanks Zaki for comment 
we have some changes on each components if is_dashboard:true ,so we should have it for all components and now by using useContext we can check it